src/components/Scenario.js

```diff
@@ -13,9 +13,18 @@ export default class Scenario extends React.Component {
   }
 
   toggle() {
+    if (this.state.collapsed === true) {
```

Comment:
I don't think the toggle action is the right place to (re)load comments. Maybe the [componentWillUpdate](http://facebook.github.io/react/docs/component-specs.html#updating-componentwillupdate) method? You could check if `!nextState.collapsed` and load comments then.